EnvButtonStartingPosition.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  EnvButtonStartingPosition.swift
+//  EnvChanger
+//
+//  Created by Nikola B Nikolov on 6.12.21.
+//
+
+import Foundation
+
+/// Static values representing positions along an axis.
+private extension CGFloat {
+    static let start: CGFloat = 0.1
```

Comment:
If the range is 0 - 1, is there are a reason for start to be 0.1 and end -> 0.9?
Is it for some padding?

Reply:
It's for some padding, yes. I'm setting the button's centre at the given position (it feels more intuitive when .center .center is passed for example), so if we put the centre at 0 or 1, part of it would be off-screen.